Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=638b250766272fcaaa0f7ed2776f58f4ac701914
Commit:     638b250766272fcaaa0f7ed2776f58f4ac701914
Parent:     a8cd925f74c3b1b6d1192f9e75f9d12cc2ab148a
Author:     Steve French <sfrench@us.ibm.com>
AuthorDate: Sat Sep 15 02:35:51 2007 +0000
Committer:  Steve French <sfrench@us.ibm.com>
CommitDate: Sat Sep 15 02:35:51 2007 +0000

    [CIFS] typo in earlier cifs_reconnect fix
    
    Signed-off-by: Steve French <sfrench@us.ibm.com>
---
 fs/cifs/connect.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/fs/cifs/connect.c b/fs/cifs/connect.c
index f58fef5..a83684d 100644
--- a/fs/cifs/connect.c
+++ b/fs/cifs/connect.c
@@ -439,7 +439,6 @@ incomplete_rcv:
 			cFYI(1, ("less than four bytes received (%d bytes)",
 			      length));
 			pdu_length -= length;
-			cifs_reconnect(server);
 			msleep(1);
 			goto incomplete_rcv;
 		}
